Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -133,9 +133,16 @@ def surround_in_condition(string, condition):
 
 
 def types_that_must_be_moved():
-    return frozenset([
+    return [
         'IPC::StreamServerConnection::Handle',
-    ])
+        'Vector<WebKit::SharedMemory::Handle>',
```

Comment:
This one is kind of unfortunate ("When we have Vector<IPC::StreamServerConnection::Handle>, do we need to add it here again?")
Maybe later we can do this in a template..